src/components/PortExplorer.svelte

```diff
@@ -1,18 +1,20 @@
 <script lang="ts">
-  import type { Port, Userstyle } from "../lib/ports";
+  import type { Port } from "../lib/ports";
   import SearchBar from "./SearchBar.svelte";
   import PortGrid from "./PortGrid.svelte";
   import Fuse from "fuse.js";
 
-  export let ports: Array<Port | Userstyle>;
-  let portGrid: Array<Port | Userstyle> | undefined = undefined;
-  let debounceTimeout: NodeJS.Timeout;
+  export let ports: Array<Port & { icon: string }>;
+  let portGrid: Array<Port & { icon: string }> | undefined = undefined;
+  let debounceTimeout: ReturnType<typeof setTimeout> | undefined;
 
   const fuse = new Fuse(ports, {
     keys: [
       { name: "key", weight: 1 },
       { name: "categories.name", weight: 0.8 },
       { name: "name", weight: 0.4 },
+      { name: "repository.current-maintainers.url", weight: 0.2 },
+      { name: "repository.current-maintainers.name", weight: 0.2 },
```

Comment:
Kinda want to keep, kinda don't

Reply:
I think we can leave this in as an undocumented feature, the weightage is so low that it really doesn't interfere with any "normal" searches.